java/system-test/system-test-cdk/src/main/java/sleeper/systemtest/cdk/SystemTestApp.java

```diff
@@ -54,17 +54,20 @@ public void create() {
 
         SystemTestBucketStack bucketStack = new SystemTestBucketStack(this, "SystemTestIngestBucket", properties, jars, getAutoDeleteS3ObjectsStack());
 
+        generateAutoStopEcsClusterTasksStack();
+
         // Control which stacks are generated by sleeperCDKApp.
         setGenerateLoggingStack(false);
         setGenerateAutoDeleteS3ObjectsStack(false);
+        setGenerateAutoStopEcsClusterTasksStack(false);
```

Comment:
It doesn't look like it's necessary to control when this stack is created like this? We did that for the S3 autodeletion because we needed to create the system test bucket before the rest of the CDK app. Is there a reason to do that for this?